src/firebase/firebase.tsx

```diff
@@ -0,0 +1,27 @@
+import firebase from 'firebase/compat/app'
+import 'firebase/compat/auth'
+import 'firebase/compat/firestore'
+
+const firebaseConfig = {
+  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
+  authDomain: process.env.REACT_APP_FIREBASE_AUTH_DOMAIN,
+  projectId: process.env.REACT_APP_FIREBASE_PROJECT_ID,
+  storageBucket: process.env.REACT_APP_FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGING_SENDER_ID,
+  appId: process.env.REACT_APP_FIREBASE_APP_ID,
+}
+
+firebase.initializeApp(firebaseConfig)
+firebase.firestore()
+
+export const rrfConfig = {
+  userProfile: 'users',
+  useFirestoreForProfile: true,
+  customAuthParameters: {
+    google: {
+      prompt: 'select_account',
+    },
+  },
+}
+
+export default firebase
```

Comment:
I'm strongly recommending not using default exports. This is quite a fragile solution due to the possibility of changing imported variable names. It sometimes leads to some typos, or, what could be worse, importing different elements than you want.

TypeScript gives you more confidence in the type of data you are operating on, and named exports gives you more confidence in using the right modules.